src/components/bottomNav/bottomNav.css

```diff
@@ -1,24 +1,20 @@
-.footer {
+.navFooter {
   display: flex;
   position: fixed;
   bottom: 0px;
-
   justify-content: space-around;
   width: 100vw;
-  /* align-content: center; */
-  /* left: 0px; */
   padding: 10px 0;
```

Comment:
thanks!
I apreciate it!:)

Just realised, that it looks weird because the space to the right side starts with the word "setting", not with the Icon...